tracing/README.md

```diff
@@ -0,0 +1,188 @@
+Distributed Tracing
+---
+
+Both Helidon SE and Helidon MP use Opentracing API for tracing
```

Comment:
If "Opentracing" here refers to the OpenTracing project (i.e., https://opentracing.io/) then change to "Open**T**racing"